config.json

```
@@ -990,6 +990,14 @@
           "conditionals"
         ]
       },
+      {
+        "slug": "square-root",
+        "name": "Square Root",
+        "uuid": "140acd81-dfa6-42ad-b2ac-bbb5917706c1",
+        "practices": [],
+        "prerequisites": [],
```

Can you double check whether there isn't an error in this suggestion? It shows the last line with a different color, not sure why.
```suggestion
        "prerequisites": [
          "numbers"
        ],
```